deployments/qat_plugin/base/intel-qat-plugin.yaml

```diff
@@ -12,6 +14,8 @@ spec:
     metadata:
       labels:
         app: intel-qat-plugin
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/intel-qat-plugin: unconfined
```

Comment:
With this becoming the default, we can drop the overlay settings

Reply:
@mythi 
Dropped the plugin's apparmor overlay and extra annotation in e2e customization.